woff2 will fail pkgconf test pipeline

If the pkgconf test pipeline is added to woff2 it will fail in the following way:

```
2024/11/12 23:16:07 INFO running step "test/pkgconf"
2024/11/12 23:16:07 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:16:07 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:16:07 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:16:07 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:16:08 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + basename /home/build/melange-out/woff2-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + dev_pkg=woff2-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + apk info -L woff2-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + grep -q ^Name: usr/lib/pkgconfig/libwoff2common.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + basename usr/lib/pkgconfig/libwoff2common.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + lib_name=libwoff2common uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + echo usr/lib/pkgconfig/libwoff2common.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + grep -q '^usr/lib/pkgconfig/libwoff2common.pc$\|^usr/share/pkgconfig/libwoff2common.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + pkgconf --exists libwoff2common uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + grep -q ^Version: usr/lib/pkgconfig/libwoff2common.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + pkgconf --modversion libwoff2common uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 INFO 1.0.2 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + grep -q ^Libs: usr/lib/pkgconfig/libwoff2common.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + pkgconf --libs libwoff2common uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 INFO -lwoff2common uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + grep -q ^Cflags: usr/lib/pkgconfig/libwoff2common.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + pkgconf --cflags libwoff2common uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 INFO uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + grep -q ^Name: usr/lib/pkgconfig/libwoff2dec.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + basename usr/lib/pkgconfig/libwoff2dec.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + lib_name=libwoff2dec uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + echo usr/lib/pkgconfig/libwoff2dec.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + grep -q '^usr/lib/pkgconfig/libwoff2dec.pc$\|^usr/share/pkgconfig/libwoff2dec.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + pkgconf --exists libwoff2dec uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN + pkgconf --print-errors --exists libwoff2dec uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN Package libbrotlidec was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN Perhaps you should add the directory containing `libbrotlidec.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 WARN Package 'libbrotlidec', required by 'libwoff2dec', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:08 INFO pod 3431e50bf22a9970b14a8a6562e21573e689377ed2d3ff41d08aa15b06dd6630 terminated
2024/11/12 23:16:08 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:16:08 INFO   workspace dir: /home/user/tmp/melange-workspace-1210026176
2024/11/12 23:16:08 INFO   guest dir: /home/user/tmp/melange-guest-4112122294
2024/11/12 23:16:08 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/woff2] Error 1
```

--- a/woff2.yaml
+++ b/woff2.yaml
@@ -1,7 +1,7 @@
 package:
   name: woff2
   version: 1.0.2
-  epoch: 1
+  epoch: 2
   description: Web Open Font Format 2 reference implementation
   copyright:
     - license: GPL-3.0-or-later
@@ -53,8 +53,12 @@ subpackages:
       - uses: split/dev
     dependencies:
       runtime:
+        - brotli-dev
         - woff2
     description: woff2 dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
 update:
   enabled: true

